LocalParks.Infrastructure/Models/ParkProfile.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoMapper;
+using LocalParks.Core.Domain;
+using LocalParks.Core.Domain.Shop;
+using LocalParks.Core.Domain.User;
+using LocalParks.Core.Models;
+using LocalParks.Core.Models.Shop;
+using System;
+
+namespace LocalParks.Infrastructure.Models
+{
+    public class ParkProfile : Profile
+    {
+        public ParkProfile()
+        {
+            CreateMap<LocalParksUser, LocalParksUserModel>().ReverseMap();
```

Comment:
automapper is cool and worth having here... but 9/10 it's easier to just write the code if someone asks why you might not want to use it.